packages/ndla-notion/src/Notion.tsx

```diff
@@ -9,80 +9,74 @@
 import React, { ReactNode } from 'react';
 import styled from '@emotion/styled';
 import { ShortText } from '@ndla/icons/common';
-import { css, InterpolationWithTheme } from '@emotion/core';
+import { InterpolationWithTheme } from '@emotion/core';
 import { createUniversalPortal } from '@ndla/util';
 import { colors } from '@ndla/core';
+import { useTranslation } from 'react-i18next';
+import Tooltip from '@ndla/tooltip';
 import NotionDialog from './NotionDialog';
 
 const BaselineIcon = styled(ShortText)`
-  content: '';
-  display: inline-block;
   position: absolute;
-  margin: calc(0.5em + 4px) auto 0;
-  left: 0;
-  color: rgba(165, 188, 211, 1);
-  transition: transform 0.1s ease;
+  margin: calc(0.5em + 1px) auto 0;
+  left: -2px;
+  color: ${colors.brand.secondary};
+  height: 1.1em;
+  width: 1.1em;
+  transition-duration: 0.5s;
 `;
-const NotionCSS = css`
-  display: inline;
-  .link {
-    background: none;
-    border: none;
-    font-family: inherit;
-    font-style: inherit;
-    line-height: 1em;
-    padding: 0 0 4px 0;
-    margin-bottom: -4px;
-    text-decoration: none;
-    color: #000;
-    position: relative;
 
-    cursor: pointer;
-
-    &:focus {
-      border-color: ${colors.brand.primary};
-      outline: none;
-      &:after {
-        transform: scale(1.4) translateY(1px);
-      }
+const StyledButton = styled.button`
+  background: none;
+  border: none;
+  font-family: inherit;
+  font-style: inherit;
+  line-height: 1em;
+  padding: 0 0 4px 0;
+  margin-bottom: -4px;
+  text-decoration: none;
+  color: #000;
+  position: relative;
+  cursor: pointer;
+  &:focus,
+  &:hover {
+    color: ${colors.brand.primary};
+    outline: none;
+    ${BaselineIcon} {
+      color: ${colors.brand.primary};
+      transform: scale(1.2, 1);
+      transform-origin: top left;
     }
   }
 `;
 
 interface Props {
   id: string;
   title: string;
-  ariaLabel: string;
   subTitle?: string;
   children?: ReactNode;
   content?: ReactNode;
   headerContent?: ReactNode;
   customCSS?: InterpolationWithTheme<any>;
   hideBaselineIcon?: boolean;
 }
-const Notion = ({
-  id,
-  ariaLabel,
-  content,
-  children,
-  title,
-  subTitle,
-  customCSS,
-  headerContent,
-  hideBaselineIcon,
-}: Props) => (
-  <span css={NotionCSS} id={id} data-notion>
-    <button type="button" aria-label={ariaLabel} className={'link'} data-notion-link>
-      {children}
-      {!hideBaselineIcon && <BaselineIcon />}
-    </button>
-
-    {createUniversalPortal(
-      <NotionDialog id={id} title={title} subTitle={subTitle} customCSS={customCSS} headerContent={headerContent}>
-        {content}
-      </NotionDialog>,
-      'body',
-    )}
-  </span>
-);
+const Notion = ({ id, content, children, title, subTitle, customCSS, headerContent, hideBaselineIcon }: Props) => {
+  const { t } = useTranslation();
+  return (
+    <span id={id} data-notion>
+      <StyledButton type="button" aria-label={title} data-notion-link aria-expanded={true}>
```

Comment:
Testet litt nå og leste på eksemplet på aria-expanded som tar for seg knapper som feks styrer modaler, akkurat som i vårt tilfelle. https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded#buttons. Her er et par tanker etter hvordan jeg tolket det.

1. Skjermleser leser nå opp "button expanded sosialisering" uten videre forklaring. Til nå har vi brukt translation med nøkkelen concept.showDescription. Forslaget mitt for å få mer kontekst er å endre denne oversettelsen til `{{title}}, begrepsforklaring. Trykk for å åpne forklaring.` eller liknende. Da kan oversettelsen tas i bruk direkte i komponenten, for jeg tror ikke vi har behov for at denne skal være forskjellig
2. aria-expanded skal være false når modalen er lukket og true når den er åpen. Default verdi blir dermed false, men verdien må oppdateres ved åpning/lukking av modal med conceptScripts fordi denne komponenten kommer til å rendres på server 😕. Kan eventuelt fjerne aria-expanded og fikse dette på et senere tidspunkt.
    - Knappen må ha en aria-controls med id som peker på NotionDialog sin id dersom dette skal implementeres.

Reply:
Enig, jeg implementerte forslaget ditt i translation. 
Fjernet også aria-expand.